addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/i18n/amazonechocontrol_de.properties

```diff
@@ -57,7 +57,7 @@ channel-type.amazonechocontrol.bluetoothMAC.label = Bluetooth Verbindung
 channel-type.amazonechocontrol.bluetoothMAC.description = MAC-Adresse des verbundenen Bluetoothger�tes

```

Comment:
This file seem to have invalid line endings (windows ?) In github it shows `^M` in red at the end of all lines in this file

Reply:
The openHAB repository uses unix line endings. Uses `git config core.eol lf` before publishing to convert automatically.

Reply:
Done